roles/osp_observability/molecule/default/verify.yml

```diff
@@ -0,0 +1,40 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Some required matching vars
+      block:
+        - name: Set config dir
+          set_fact:
```

Comment:
ansible.builtin.set_fact? Not sure how picky we are about using the FQCN

Reply:
We can debate it on a PR to this file? https://github.com/infrawatch/osp-observability-ansible/blob/master/.ansible-lint